oak/server/dev/dev_oak_manager.h

```diff
@@ -39,14 +41,7 @@ class DevOakManager final : public Manager::Service {
                                     TerminateApplicationResponse* response) override;
```

Comment:
(for l.31, somewhat-pre-existing) Maybe add a class comment explaining what this class does (it now forwards all its functionality on to a `DevOakLoader`, so I think the key thing is that it implements the gRPC service interface for `Manager`)

Reply:
Done